src/common/constants.ts

```diff
@@ -12,5 +12,5 @@ export const SERVICES: Record<string, symbol> = {
   CONFIG: Symbol('Config'),
   TRACER: Symbol('Tracer'),
   METER: Symbol('Meter'),
-};
+} satisfies Record<string, symbol>;
```

Comment:
I love the idea of 
```typescript
satisfies Record<string, symbol>;
```  
but the type definition should be removed if it's used.
![image](https://github.com/user-attachments/assets/c25cb77d-da5f-462d-87ae-eb2395f55e7c)

Reply:
type definition removed